src/main/java/org/dependencytrack/util/CacheStampedeBlocker.java

```diff
@@ -140,6 +140,7 @@ public CacheStampedeBlocker(String cacheName, int nbBuckets, boolean blockCompet
         RetryConfig config = RetryConfig.custom()
                 .maxAttempts(this.nbRetryMax)
                 .intervalFunction(intervalWithCustomExponentialBackoff)
+                .ignoreExceptions(ignoredExceptions)
```

Comment:
Well I suppose it would be safer. For repo meta I knew what I was going for and trapped all runtime exceptions but we can't expect every contributor to be aware of that feature.
I'll inverse it 👍